apps/whale-api/src/module.api/consortium.service.ts

```diff
@@ -0,0 +1,93 @@
+import { Injectable } from '@nestjs/common'
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+import { DeFiDCache } from './cache/defid.cache'
+import { SemaphoreCache } from '@defichain-apps/libs/caches'
+import { AssetBreakdownInfo, MemberInfo } from '@defichain/whale-api-client/dist/api/consortium'
+import BigNumber from 'bignumber.js'
+
+@Injectable()
+export class ConsortiumService {
+  constructor (
+    protected readonly rpcClient: JsonRpcClient,
+    private readonly defidCache: DeFiDCache,
+    protected readonly cache: SemaphoreCache
+  ) {}
+
+  private pushToAssetBreakdownInfo (memberInfo: MemberInfo[], tokens: any[], assetBreakdownInfo: AssetBreakdownInfo[], key: string, value: string): void {
+    const tokenId = key.split('/')[4]
+    const memberId = key.split('/')[5]
+    const member = memberInfo.find(am => am.id === memberId)
+    const type = key.split('/')[6] as 'burnt' | 'minted' | 'supply'
+    const token = tokens?.find(t => t.id === tokenId)
+    const val = new BigNumber(value).toFixed(+token.decimal.toString())
+
+    const memberObj = {
+      id: memberId,
+      name: member?.name ?? '',
+      backingAddress: member?.backingAddress ?? '',
+      minted: '-1',
+      backed: '-1',
```

Comment:
Backed amount is not fetched at the moment as it will be added in a future iteration.